src/ensembl/src/content/app/browser/track-panel/trackPanelSelectors.ts

```diff
@@ -1,21 +1,30 @@
 import { RootState } from 'src/store';
 import { getBrowserActiveGenomeId } from '../browserSelectors';
 import { TrackType } from './trackPanelConfig';
+import { TrackPanelStateForGenome } from './trackPanelState';
+
+const getTrackPanelStateProp = (
```

Comment:
getTrackPanelProperty?

Reply:
Also, do I understand correctly that this function is only used in this selector?

Perhaps instead of casting return value of this to various types it will be better to just have `getActiveTrackPanel` (which returns `TrackPanelStateForGenome` or null) and then, in other selectors, pick relevant properties out of that object?

Reply:
>Also, do I understand correctly that this function is only used in this selector?

Yes

>Perhaps instead of casting return value of this to various types it will be better to just have getActiveTrackPanel (which returns TrackPanelStateForGenome or null) and then, in other selectors, pick relevant properties out of that object?

That will be neater